app/views/posts/show.html.erb

```diff
@@ -7,3 +7,16 @@
 </div>
 
 <%= render "shared/comment_box", comments: @post.comments %>
+
+<div class="form-container">  
+  <%= form_with model: Comment.new, url: user_post_comments_path(@post.author, @post), local: true do |form| %>
+    <%= form.label :text %>
+    <%= form.text_area :text %>
+
+    <%= form.submit "Add Comment", class: "form-btn" %>
+  <% end %>
+
+  <%= form_with model: Like.new, url: user_post_likes_path(@post.author, @post), local: true do |form| %>
+    <%= form.submit "Like", class: "form-btn" %>
+  <% end %>
+</div>
```

Comment:
Good job so far. Everything is looking fine. However, according to the project requirements, there should be a form in the `views/comments/new.html.erb` file which accepts input for creating comments which is not present at the moment. Kindly create the file (`views/comments/new.html.erb`) and add this input for creating comments

Reply:
Hello @jerryowusu could you elaborate a bit more. I've looked over the requirements and I don't see any requirements stating that the comments form must be created a separate view.


![image](https://github.com/jlvFlores/RoR-blog-app/assets/65242260/abc5d5c4-b0f5-4c40-af0a-04a989e7c04d)